index.html

```diff
@@ -46,4 +46,113 @@ <h3 class="presentation">I'm a Software Developer</h3>
+      <li class="tool_card">Ruby</li>
+    </ul>
+    <a href="#" class="see_project_card">See Project</a>
+  </div>
+  <div class="project_card card4">
+    <h3 class="card_title">Professional Art<br> Printing Data</h3>
+    <p class="card_description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tools_list_card">
+      <li class="tool_card">HTML</li>
+      <li class="tool_card">Bootstrap</li>
+      <li class="tool_card">Ruby</li>
+    </ul>
+    <a href="#" class="see_project_card">See Project</a>
+  </div>
+  <div class="project_card card5">
+    <h3 class="card_title">Professional Art<br> Printing Data</h3>
+    <p class="card_description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tools_list_card">
+      <li class="tool_card">HTML</li>
+      <li class="tool_card">Bootstrap</li>
+      <li class="tool_card">Ruby</li>
+    </ul>
+    <a href="#" class="see_project_card">See Project</a>
+  </div>
+  <div class="project_card card6">
+    <h3 class="card_title">Professional Art<br> Printing Data</h3>
+    <p class="card_description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tools_list_card">
+      <li class="tool_card">HTML</li>
+      <li class="tool_card">Bootstrap</li>
+      <li class="tool_card">Ruby</li>
+    </ul>
+    <a href="#" class="see_project_card">See Project</a>
+  </div>
+  </section>
+  <section class="about_me">
+    <div class="about_me_intro">
+      <h2 class="about_me_title">About me</h2>
+      <p class="about_me_description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <a href="#" class="about_me_button">Get my resume</a>
+    </div>
+    <hr class="about_me_break">
+    <ul class="about_me_tools_list">
+      <li class="about_me_tool Language">Language</li>
+        <ul class="unit_tool_list">
+          <li class="language">JavaScript</li>
+          <li class="language">Ruby</li>
+          <li class="language">HTML</li>
+          <li class="language">CSS</li>
+        </ul>
+      <li class="about_me_tool Frameworks">Frameworks</li>
+        <ul class="unit_tool_list">
+          <li class="framework">React.js</li>
+          <li class="framework">Ruby on Rails</li>
+          <li class="framework">RSpec</li>
+          <li class="framework">Capybara</li>
+        </ul>
+      <li class="about_me_tool Skills">Skills</li>
+        <ul class="unit_tool_list">
+          <li class="skill">Database Management</li>
+          <li class="skill">Version Control</li>
+          <li class="skill">CLI</li>
+          <li class="skill">API Design</li>
+          <li class="skill">Web Design</li>
+        </ul>  
+    </ul>
+
```

Comment:
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/251997323-883ebf22-990a-4abd-98c0-765962638103.png)|![image](https://user-images.githubusercontent.com/99715640/251997433-3e7a92ca-8d65-4d76-b419-e339871fea32.png)|

- [ ] Kindly ensure that you stick to the Figma design as much as you can in implementing the spacing/margin/padding issues earmarked in the figure above.